DQM/Integration/python/config/FrontierCondition_GT_cfi.py

```diff
@@ -1,3 +1,3 @@
 import FWCore.ParameterSet.Config as cms
 from Configuration.StandardSequences.FrontierConditions_GlobalTag_cff import *
-GlobalTag.globaltag = "111X_dataRun3_HLT_GEMRecoGEO_v1"
+GlobalTag.globaltag = "111X_dataRun3_Express_v4"
```

Comment:
@christopheralanwest, to be fair, a new Run3 HLT Global Tag is needed here (but I haven't found any without the records removed in #31729 .

Reply:
I created a new GT, `111X_dataRun3_HLT_v3`, with the obsolete records removed: https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/111X_dataRun3_HLT_v2/111X_dataRun3_HLT_v3

Reply:
thanks, fixed now.